web/templates/about_client_program/show.html.eex

```diff
@@ -0,0 +1,86 @@
+  <div class="row">
+    <div class="
+      col-xs-12
+      col-sm-offset-1 col-sm-10
+      col-md-offset-2 col-md-8
+      col-lg-offset-2 col-lg-8
+    ">
+
+      <div class="row middle-xs page-header">
+        <div class="col-xs-6 col-sm-9 col-md-10 page-title">
+          About <%= @program_data.name %>
+        </div>
+      </div>
+
+      <div class="page">
+          <%= CoursePlanner.SharedView.card do %>
+            <%= CoursePlanner.SharedView.card_content vpadding: true do %>
+              <div class="row row--hspace row--vspace middle-xs">
+                <%= if not is_nil(@program_data.logo) do %>
```

Comment:
`if` statement in Elixir checks for truthy values, meaning anything not `nil` or `false` is considered true, so you can check simply like this: `if @program_data.logo do`